3-enrich/stream-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/AbstractSource.scala

```diff
@@ -237,13 +235,20 @@ abstract class AbstractSource(
       m <- MaxRecordSize
     } yield AbstractSource.oversizedSuccessToFailure(value, m) -> key
 
-
-    val anonymizedSuccesses = smallEnoughSuccesses.map({case (event: String, partition: String, _) => (event, partition)})
-    val piiSuccesses = smallEnoughSuccesses.flatMap({case (_, partition: String, pii: Option[String]) => pii.map((_, partition))})
+    val anonymizedSuccesses = smallEnoughSuccesses.map {
+      case (event: String, partition: String, _) => (event, partition)
+    }
+    val piiSuccesses = smallEnoughSuccesses.flatMap {
+      case (_, partition: String, pii: Option[String]) => pii.map((_, partition))
+    }
     val successesTriggeredFlush = sink.get.map(_.storeEnrichedEvents(anonymizedSuccesses))
-    val piiTriggeredFlush = piiSink.get.map(_.storeEnrichedEvents(piiSuccesses))
-    val failuresTriggeredFlush = badSink.get.map(_.storeEnrichedEvents(failures ++ sizeBasedFailures))
-    if (successesTriggeredFlush == Some(true) || failuresTriggeredFlush == Some(true) || piiTriggeredFlush == Some(true)) {
+
+    val piiTriggeredFlush       = piiSink.get.map(_.storeEnrichedEvents(piiSuccesses))
+    val failuresTriggeredFlush =
+      badSink.get.map(_.storeEnrichedEvents(failures ++ sizeBasedFailures))
+    if (successesTriggeredFlush == Some(true) ||
```

Comment:
Linter inside my IDE usually proposes to change these to `a.contains(true) || b.contains(true)`. Just a nit, not requirement.